docs/cookbooks/creating_a_derived_dataset.md

```diff
@@ -284,6 +285,7 @@ For our example:
 
 3. You will be notified when swapping is complete.
 
+**Note**.  Please announce in the #data-platform-infra-wg Slack channel before deleting any tables in the `backfill_staging_derived` dataset since it may cause issues in the workflow.
```

Comment:
I'm not sure what a good approach here is. Restricting delete access would be ideal... I think this could also be a good thing for the person on triage to handle.

Reply:
Yeah, maybe this just becomes a known fix for failed backfills during triage, depending on why the backfill failed. Some of this might make it into the DAG docs.